service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/CaseManagementOrderCaseLinkNotificationHandler.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.hmcts.reform.fpl.handlers;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.fpl.config.LocalAuthorityNameLookupConfiguration;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.Representative;
+import uk.gov.hmcts.reform.fpl.model.event.EventData;
+import uk.gov.hmcts.reform.fpl.service.InboxLookupService;
+import uk.gov.hmcts.reform.fpl.service.RepresentativeService;
+import uk.gov.hmcts.reform.fpl.service.email.NotificationService;
+import uk.gov.hmcts.reform.fpl.service.email.content.CaseManagementOrderEmailContentProvider;
+
+import java.util.List;
+import java.util.Map;
+
+import static org.apache.commons.lang.StringUtils.isNotBlank;
+import static uk.gov.hmcts.reform.fpl.NotifyTemplates.CMO_ORDER_ISSUED_CASE_LINK_NOTIFICATION_TEMPLATE;
+import static uk.gov.hmcts.reform.fpl.enums.RepresentativeServingPreferences.DIGITAL_SERVICE;
+
+@Component
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class CaseManagementOrderCaseLinkNotificationHandler {
+    private final ObjectMapper objectMapper;
+    private final InboxLookupService inboxLookupService;
+    private final NotificationService notificationService;
+    private final RepresentativeService representativeService;
+    private final LocalAuthorityNameLookupConfiguration localAuthorityNameLookupConfiguration;
+    private final CaseManagementOrderEmailContentProvider caseManagementOrderEmailContentProvider;
+
+    public void sendCMOCaseLinkNotifications(final EventData eventData) {
+        sendCMOCaseLinkNotificationForLocalAuthority(eventData);
+        sendCMOCaseLinkNotificationToRepresentatives(eventData);
+    }
+
+    private void sendCMOCaseLinkNotificationForLocalAuthority(final EventData eventData) {
+        final String localAuthorityName = localAuthorityNameLookupConfiguration.getLocalAuthorityName(
```

Comment:
A bit inconsistent when you choose to use final, `email` shouldn't change either so should it be final as well? To that effect so should the map, maybe all or none?